utils/fork-tree/src/lib.rs

```diff
@@ -32,7 +32,7 @@ pub enum Error<E> {
 	UnfinalizedAncestor,
 	/// Imported or finalized node that is an ancestor of previously finalized node.
 	Revert,
-	/// Error throw by client when checking for node ancestry.
+	/// Error thrown by user when checking for node ancestry.
 	Client(E),
```

Comment:
Maybe rename this to `Ancestry(E)`? Or something else if someone has a better suggestion.